src/COVID19/model.py

```diff
@@ -507,16 +536,16 @@ def one_time_step_results(self):
             self.c_model, covid19.RECOVERED
         )
         results["hospital_admissions"]  = covid19.utils_n_daily(
-            self.c_model, covid19.TRANSITION_TO_HOSPITAL, self.c_model.time
+            self.c_model, covid19.GENERAL, self.c_model.time
         )
```

Comment:
Rob, these require the old, extra transitions that Bryn added to work. Should we add those back int?